android/src/main/java/com/thebluealliance/androidclient/helpers/PitLocationHelper.java

```diff
@@ -0,0 +1,201 @@
+package com.thebluealliance.androidclient.helpers;
+
+import com.thebluealliance.androidclient.R;
+import com.thebluealliance.androidclient.models.APIStatus;
+
+import android.content.Context;
+import android.preference.PreferenceManager;
+import android.util.JsonReader;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Encapsulation of special-cased stuff for showing pit locations at 2016 Champs
+ */
+public class PitLocationHelper {
+
+    private static final String LAST_UPDATED_PREF_KEY = "cmp_pit_locations_update_time";
+    private static final String PIT_LOCATIONS_FILENAME = "2016_pit_locations.json";
+
+    private static Map<String, TeamPitLocation> sTeamLocationCache = new HashMap<>();
+
+    public static boolean shouldShowPitLocation(Context context, String teamKey) {
+        Date champsStart = new GregorianCalendar(2016, GregorianCalendar.APRIL, 25).getTime();
```

Comment:
Can you add a `TODO` here to improve in the future. I like the idea of reading an event key from the status endpoint, and only showing if it's live, or something